src/Controller/Organization/PlanningController.php

```diff
@@ -39,21 +40,33 @@ public function __construct(UserRepository $userRepository, CommissionableAssetR
     public function __invoke(Request $request): Response
     {
         $data = [
-            'from' => new \DateTimeImmutable('monday'),
-            'to' => (new \DateTimeImmutable('monday'))->add(new \DateInterval('P1W')),
+            'from' => new \DateTimeImmutable('monday this week'),
+            'to' => (new \DateTimeImmutable('monday this week'))->add(new \DateInterval('P1W')),
             'volunteer' => true,
             'volunteerEquipped' => true,
             'volunteerHideVulnerable' => true,
             'asset' => true,
+            'submit' => '',
         ];
 
+        if ($request->query->has('preselect')) {
```

Comment:
I would have sent GET from, to and organization[] parameters instead of a custom `preselect` with two values

Reply:
En fait c'est chiant a faire, vu que le formulaire est en get, il faut envoyer toutes les variables si tu veux que ça marche :o